pkgs/by-name/op/opentelemetry-cpp/package.nix

```diff
@@ -48,9 +47,7 @@ stdenv.mkDerivation (finalAttrs: {
     nlohmann_json
   ];
 
-  propagatedBuildInputs =
-    [ abseil-cpp ]
-    ++ lib.optionals enableGrpc [
+  propagatedBuildInputs = lib.optionals enableGrpc [
```

Comment:
There is a nixfmt error (which I introduced via a previous suggestion). Otherwise it looks good to me.